google-cloud-spanner/src/main/java/com/google/cloud/spanner/Value.java

```diff
@@ -218,7 +224,82 @@ public static Value pgJsonb(@Nullable String v) {
   }
 
   /**
-   * Returns a {@code BYTES} value.
+   * Return a {@code PROTO} value.
+   *
+   * @param v Not null Proto message.
+   */
+  public static Value protoMessage(AbstractMessage v) {
+    Preconditions.checkNotNull(v, "Use protoMessage(ByteArray, Descriptor) for null values.");
```

Comment:
nit: change the error message to include a specific example (e.g. use `protoMessage((ByteArray) null, Descriptor.getDefaultInstance())` for null vaues.